README.md

```diff
@@ -1,26 +1,113 @@
-# README
+# Budget App
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+![Microverse](https://img.shields.io/badge/Microverse-blueviolet)
 
-Things you may want to cover:
+> A mobile web application where you can manage your budget
 
-* Ruby version
+Home page |
+----------|
+![Screenshot from 2021-12-04 13-28-14](https://user-images.githubusercontent.com/65068771/144707791-ac4b41f3-2af1-4945-8c65-627dc30c6fca.png)
 
-* System dependencies
+  
 
-* Configuration
+A mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
 
-* Database creation
+## Built With
 
-* Database initialization
+- Ruby on Rails
+- PostgreSQL
```

Comment:
- [ ] I would recommend deploying the project and it must be accessible online. I heard that you have already deployed 5 websites on Heroku, but I would suggest that if any of the projects is frontend please move it to netlify or if any application is not that important please delete and kindly deploy this application online. As this is a requirement please deploy it

Reply:
- [ ] Hello @Dipeshtwis  kindly all of the 5 projects deployed on Heroku are backEnds and are in use. I would have deleted them but they are in use. Kindly where else should I deploy my app??

Reply:
- [ ] I am trying to use GitHub for that bu is not working?